app/views/print/terms_of_deposit.html.erb

```diff
@@ -0,0 +1,9 @@
+<script language='javascript'>
+  window.onload = function() { 
+    document.getElementById("print-link").style.display='none';
+    window.print(); 
+  }
+</script>
+<link rel="stylesheet" type="text/css" media="print" href="bootstrap.min.css"> 
```

Comment:
Why do we need this and where is it coming from?

Reply:
removed, leftover cruft from experimentatio -- skipping the layout is enough to give us a clean page for printing